RELEASE.md

```diff
@@ -1,39 +1,51 @@
 # Releasing the Jaeger Operator for Kubernetes
 
-1. Update Jaeger version in `versions.txt`
+Steps to release a new version of the Jaeger Operator:
 
-1. Make sure the new version is present at `pkg/upgrade/versions.go`
 
-1. Prepare a changelog since last release. Get the `OAUTH_TOKEN` from (Github)[https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token] and select `repo:status` scope.
+1. Change the `versions.txt `so that it lists the target version of the Jaeger (if it is required). Don't touch the operator version it will be changed automatrically in the next ste.
 
-    ```
-    OAUTH_TOKEN=... make changelog
-    ```
+1. Run `make prepare-release VERSION=1.25.0`, using the operator version that will be released.
+
+1. Prepare a changelog since last release. 
 
-1. Commit version change and changelog and create a pull request:
+1. Commit the changes and create a pull request:
 
    ```
    git commit -sm "Preparing release v1.25.0"
    ```
 
-1. Tag and push
+1. Once the changes above are merged and available in `master` tag it with the desired version, prefixed with `v`, eg. `v1.25.0`
 
     ```
-    git checkout master ## it's only possible to release from master for now!
-    git tag release/v1.25.0
-    git push git@github.com:jaegertracing/jaeger-operator.git release/v1.25.0
+    git checkout master
+    git tag v1.25.0
+    git push git@github.com:jaegertracing/jaeger-operator.git v1.25.0
     ```
 
-1. Wait until release CI job finishes and then pull the changes:
+1. The GitHub Workflow will take it from here, creating a GitHub release and publishing the images
```

Comment:
the following git pull can be removed. It's documented later when updating the operator hub